### PR TITLE
added NMR MTBLS1 workflow

### PR DESCRIPTION
**Pull request by korseby**
Hi @pcm32 @rsalek and the other NMR people,

I created a first running version of the NMR MTBLS1 workflow to be included in the next release. Please merge.

All the best, *Kristian

### CODE REVIEW

File: external-datatypes/nmrml_datatype.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from galaxy.datatypes.sniff import *
+from galaxy.datatypes.binary import *
+from galaxy.datatypes.tabular import *
+from galaxy.datatypes.xml import *
+from galaxy.datatypes.proteomics import *
+
+log = logging.getLogger(__name__)
+
+
+class MzML(ProteomicsXml):
```
Comment (pcm32) [inline external-datatypes/nmrml_datatype.py]: Why does this say MzML??

File: tools/phenomenal/nmr/rnmr1d.xml [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
   <description>Preprocessing RAW NMR data with macro-commands.</description>
   <command><![CDATA[
-    echo $infile;
-    echo $macrofile;
-    ls -al $infile;
     mkdir -p /data/input;
     mkdir -p /data/output;
-    /opt/rnmr1d/exec/Rnmr1D --zip $infile --proccmd $macrofile --outnorm PQN --cpu 8 --outdir /data/output | tee /opt/output/stdout.log;
-    cp -f /data/output/data_matrix.txt $outfile;
+    /opt/rnmr1d/exec/Rnmr1D --zip $infile --proccmd $macrofile --outnorm PQN --cpu 8 --outdir /data/output 2>&1;
```
Comment (pcm32) [inline tools/phenomenal/nmr/rnmr1d.xml]: Would this work without the explicit path?

**Comment by pcm32** [comment]
I PR-ed some changes for the codacy checks, please accept that pull and they will appear here. Can you then test that the galaxy instance starts well with those changes (unfortunately here at the conference and hotel internet is not fast at all enough to build containers :-( ). Thanks!

**Comment by korseby** [comment]
Hi @pcm32 

Thanks for spotting it!
`class MzML(ProteomicsXml):` has to be changed to `class NmrML(ProteomicsXml):`.

`/opt/rnmr1d/exec/Rnmr1D` has to be in the absolute path, because the structure of Rnmr1D requires it. Please note that `2>&1` is also required for Galaxy to exit without an error code.

Can you make the required changes? I still have 7 hours left at work today. ;)

**Comment by pcm32** [comment]
Why did we close this?

**Pull request closed without merging**